app/services/hyrax/listeners/object_lifecycle_listener.rb

```diff
@@ -14,14 +14,24 @@ def on_object_deleted(event)
       ##
       # @param event [Dry::Event]
       def on_object_deposited(event)
+        return unless run_jobs(event)
         ContentDepositEventJob.perform_later(event[:object], event[:user])
       end
 
       ##
       # @param event [Dry::Event]
       def on_object_metadata_updated(event)
+        return unless run_jobs(event)
         ContentUpdateEventJob.perform_later(event[:object], event[:user])
       end
+
+      private
+
+      def run_jobs(event)
+        # TODO: Jobs should run for all model types, but currently fails for
```

Comment:
i think the scope of this listener isn't all model types, but "PCDM Objects". in Hydra Works, we only account for "Work and "FileSet" as types of Objects.

what am i missing?